### ISSUE TITLE
Cannot use any specific pico-w libraries, include errors

### ISSUE THREAD

**Issue by diminDDL** [raspberrypi/pico-sdk#916]
After updating the pico-sdk to the latest version (to use the pico w) by running `git pull` I tried to compile a simple example to see if everything worked. And it didn't. Any time I try to include for example `pico_cyw43_arch` in my `target_link_libraries` and try to build something I get `[build] /usr/lib/gcc/arm-none-eabi/9.2.1/../../../arm-none-eabi/bin/ld: cannot find -lpico_cyw43_arch`. Did I update the sdk wrong? Why can't it find it. If I navigate to the `PICO_SDK_PATH/src/rp2_common/pico_cyw43_arch/` manually with the file explorer I can see the files and everything. So, it is there, cmake just can't find it.

**Comment by kilograham** [comment]
you need to use `PICO_BOARD=pico_w`

**Comment by diminDDL** [comment]
Can you clarify where exactly should I add this?

**Comment by lurch** [comment]
See section 2.2 of https://datasheets.raspberrypi.com/picow/connecting-to-the-internet-with-pico-w.pdf

**Comment by kilograham** [comment]
TLDR pass `-DPICO_BOARD=pico_w` to CMake, or add `PICO_BOARD=pico_w` to your environment
